packages/syntax-core/src/Button/Button.tsx

```diff
@@ -107,9 +107,9 @@ const Button = ({
 }): ReactElement => {
   const contextProps = useContext(ButtonGroupContext);
 
-  const size = contextProps?.size || sizeProp;
-  const disabled = contextProps?.disabled || disabledProp;
-  const fullWidth = contextProps?.fullWidth || fullWidthProp;
+  const size = contextProps?.size ?? sizeProp;
```

Comment:
@somethiiing I don't think we can use `??` in this case since the button should not be disabled when `disabled` is `false`

Reply:
fixed by adding comments + disabling the warnings for using `||`